src/Database.cpp

```diff
@@ -127,11 +131,35 @@ void Database::checkMetaTable() {
     int dbVersion = sqlite3_column_int(stmt, 0);
     sqlite3_finalize(stmt);
 
-    if (dbVersion != DATABASE_VERSION) {
-        // we should be handling migrations here in the future
-        std::cout << "[FATAL] DB Version doesn't match Server Build" << std::endl;
+    if (dbVersion > DATABASE_VERSION) {
+        std::cout << "[FATAL] Server Build is incompatible with DB Version" << std::endl;
         exit(1);
     }
+
+    while (dbVersion != DATABASE_VERSION){
+        // db migrations
+        std::cout << "[INFO] Migrating Database to Version " << dbVersion + 1 << std::endl;
+
+        std::string path = "sql/migrations/" + std::to_string(dbVersion) + ".sql";
```

Comment:
I don't think we need a nested directory. Let's either call it `sql` or `migrations`.

Or, hmm. I'm not sure. We'll discuss it.